tests/unit/sagemaker/image_uris/test_smp_v2.py

```diff
@@ -38,6 +39,11 @@ def test_smp_v2(load_config):
                         if "2.1" in version or "2.2" in version or "2.3" in version:
                             cuda_vers = "cu121"
 
+                        if "2.3.1" == version:
+                            py_version = "py311"
+
+                        print(version, py_version)
```

Comment:
Remove this print

Reply:
Oops, accidentally left that in. Removing now

Reply:
Or wait, it is merged already, is it worth making a new PR for it?